### PR TITLE
Updated the readme for local development

### PR DESCRIPTION
**Pull request by drewpotter**
Slight changes to the readme for local development.

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -34,11 +34,14 @@ March 2017 - Updated the functionality of the filters on the Resources and Data
...
   bundle install (see troubleshooting if you have issues)
+  mv config/database.yml.example config/database.yml
+  rake db:create
   rake db:migrate
   rake db:seed
   HOSTNAME="localhost:3000" rake unepwcmc:setup
-  mv config/database.yml.example config/database.yml
-  
+  rails generate comfy:cms
+  rake db:migrate
```
Comment (Levia) [inline README.rdoc]: We probably don't need an extra `db:migrate` here. There's one already after `db:create`

File: README.rdoc [hunk 2]
```diff
@@ -34,11 +34,14 @@ March 2017 - Updated the functionality of the filters on the Resources and Data
...
 
   bundle install (see troubleshooting if you have issues)
+  mv config/database.yml.example config/database.yml
+  rake db:create
   rake db:migrate
   rake db:seed
   HOSTNAME="localhost:3000" rake unepwcmc:setup
-  mv config/database.yml.example config/database.yml
-  
+  rails generate comfy:cms
```
Comment (Levia) [inline README.rdoc]: We might want to be more specific about this as this generates some extra lines of codes and files that might need to be deleted. Such as extra lines in the `routes.rb` file about the cms paths which are already existing, or the genertion of the `app/assets/stylesheets/comfy/admin/cms/custom.sass` file; there is already one of this with different extention and if both exist the system will generate an exception saying it doesn't know which one to use.
This line will also potentially overwrite the `initializer.rb` file if allowed to.

File: README.rdoc (hunk 1 above)
Comment (drewpotter) [reply README.rdoc]: Thanks @Levia I will fix this.

File: README.rdoc (hunk 2 above)
Comment (drewpotter) [reply README.rdoc]: Thanks @Levia I need to update this, as in the end I just used the production database and everything just worked.

Comment (Levia) [reply README.rdoc]: Yep, you can also mention that getting a db dump si recommended

File: README.rdoc [hunk 3]
```diff
@@ -33,16 +33,13 @@ March 2017 - Updated the functionality of the filters on the Resources and Data
...
   mv config/database.yml.example config/database.yml
   rake db:create
-  rake db:migrate
-  rake db:seed
-  HOSTNAME="localhost:3000" rake unepwcmc:setup
-  rails generate comfy:cms
-  rake db:migrate
 
-Amend HOSTNAME as required and provide your database settings in database.yml.
+The easiest path is to get a dump of the database and to import that into PostgreSQL. Now simply run:
```
Comment (Levia) [inline README.rdoc]: This is fine, but I would still leave the previous instruction for reference as I think include some useful info about comfy and some custom rake tasks.
You can keep both by specifying which one is preferred and which one was the default way etc.

Comment (drewpotter) [reply README.rdoc]: Many thanks @Levia I hope it is alright now?

**Pull request merged**